src/NUnitFramework/tests/Constraints/ThrowsConstraintTests.cs

```diff
@@ -119,6 +119,7 @@ public void SucceedsWithGoodValues(object value)
         }
 
         [Test, TestCaseSource("FailureData")]
+        [SetUICulture("en-US")]
         public void FailsWithBadValues(object badValue, string message)
```

Comment:
Failure reason: Test asserts a constant English exception message that doesn't match the localized exception message.